index.bs

```diff
@@ -688,6 +702,10 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
 1. Let |clientDataHash| be the [=hash of the serialized client data=] represented by |clientDataJSON|.
 
+1. If the <code>|options|.{{CredentialCreationOptions/signal}}</code> is [=present=] and its 
+    [=AbortSignal/aborted flag=] is set to true, return [=a promise rejected with=] an "{{AbortError}}" 
```

Comment:
we cannot return a promise here because we're just a synchronous algorithm; the Promise resolution/rejection is handled by `navigator.credentials.create()`.  see https://github.com/w3c/webappsec-credential-management/pull/100

[webappsec-credential-management/#algorithm-create](https://w3c.github.io/webappsec-credential-management/#algorithm-create) already is checking for `options.signal`'s aborted flag being set, but may need to also be doing it in the "in parallel" section.   maybe we need to have webauthn's `[[Create]]()` alg simply `return` as quickly as possible if an abort situation arises, and then have [webappsec-credential-management/#algorithm-create](https://w3c.github.io/webappsec-credential-management/#algorithm-create) check for abort right after webauthn's `[[Create]]()` alg returns to it ?

Reply:
Does the note in the abort section help explain this? The reality is abort is a new concept to web platform APIs and our current spec languages don't handle this that well. This is why the section mentions we are checking the aborted flag three times to "fake" the effect of rejecting immediately.

Reply:
I'm not sure.  this will ostensibly get review from folks who are more familiar with specifying this new abort hotness and we will see.